pkgs/games/dwarf-fortress/game.nix

```diff
@@ -135,8 +144,18 @@ stdenv.mkDerivation {
               @executable_path/libs/libstdc++.6.dylib \
       $exe
   '' + ''
-    # Store the new hash
+    ls -al $out
+    runHook postInstall
+  '';
+
+  fixupPhase = ''
+    runHook preFixup
+    runHook postFixup
```

Comment:
postFixup should be at the end

Reply:
Hmm... fixed it with an exit trap. It actually needs to run after the last fixup due to autoPatchelf.